src/ui/mainwindow.h

```diff
@@ -305,6 +304,7 @@ class MainWindow : public QMainWindow, public PlatformInterface {
   DeviceView* device_view_;
   SongInfoView* song_info_view_;
   ArtistInfoView* artist_info_view_;
+  QueueManager* queue_manager_;
```

Comment:
Yes, `unique_ptr` was deleting this automatically in `MainWindow::~MainWindow()`